### PR TITLE
FINERACT-1191: Finceract Documentation in Asciidoc

### PR DESCRIPTION
**Pull request by vidakovic**
FINERACT-1191

## Description

Translated everything under the docs folder into Asciidoc format. The JSON request specs are now under the resources folder (didn't know what else to do with them or if they are actually still required). References to Mifos replaced with Fineract (wherever I found them). Order of chapters is somewhat arbitrary, just made a choice that didn't seem too odd. Find attached a first sample for the PDF output.

[index.pdf](https://github.com/apache/fineract/files/5451935/index.pdf)

### CODE REVIEW

**Comment by ptuomola** [comment]
@vidakovic looks like the build is failing due to some invalid option..?

**Comment by vidakovic** [comment]
@ptuomola my bad... "--fail-fast" is a parameter on "integrationTest" and not on the whole Gradle command. Fixed it.

File: fineract-doc/src/docs/en/colophon.adoc [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+//ifdef::backend-pdf[]
+//{empty} +
+//endif::[]
+
+.Authors
+|===
+^| Name ^| Email
+
+| Michael Vorburger | mailto:michael@vorburger.ch[]
```
Comment (vorburger) [inline fineract-doc/src/docs/en/colophon.adoc]:
```suggestion
| Michael Vorburger | mailto:mike@vorburger.ch[]
```

**Comment by vorburger** [comment]
Interesting. FYI build failure here should be addressed by my #1474.

**Comment by vidakovic** [comment]
@ptuomola @vorburger FYI: found the solution for the failing build... easy, just Ghostscript was missing (for PDF doc build)

**Review by vorburger — APPROVED** [review]
I support this (great work @vidakovic - thanks!!). It just needs a rebase after merge of #1465, and best also to FIRST merge #1489 and only THEN rebase this one. -- PS: This also made me think of yet another :smile: new idea, see FINERACT-1253.

**Comment by vidakovic** [comment]
@vorburger #1465 and #1489 are included here. Just waiting for the build to finish...

**Pull request merged**